includes/admin/views/access-plans/metabox.php

```diff
@@ -11,6 +11,14 @@
  */
 
 defined( 'ABSPATH' ) || exit;
+?>
+
+<?php
+// get available checkout redirection types.
+$checkout_redirection_types = llms_get_checkout_redirection_types();
+
+$checkout_redirection_types['self'] = sprintf( $checkout_redirection_types['self'], ( ! $course ) ? __( 'Membership', 'lifterlms' ) : __( 'Course', 'lifterlms' ) );
```

Comment:
Per comment above the `sprintf()` could be removed from here and done within the `llms_get_checokut_redirection_types()` func.

Reply:
moved